manifests/config.pp

```diff
@@ -37,4 +37,18 @@
     mode   => '0750',
   }
 
+  file { $pulpcore::allowed_import_path:
+    ensure => directory,
+    owner  => $pulpcore::user,
+    group  => $pulpcore::group,
+    mode   => '0760',
```

Comment:
760 is a very unusual permission to have. I'd either expect `0770` or `0750`.

Reply:
The 6 means the group can read and write but not execute.  thanks to @chris1984 for showing me http://permissions-calculator.org/

Reply:
I know that, but why would you not execute it? To `cd` to a directory you need execute permissions. You can't create a file in it. It doesn't make sense to me and you get very weird errors:
```console
$ ls -ld /tmp/bla
drwxrw-rw-. 2 root root 60 Dec  8 18:10 /tmp/bla
$ sudo rm -rf /tmp/bla
$ sudo mkdir -m 0766 /tmp/bla
$ ls -ld /tmp/bla
drwxrw-rw-. 2 root root 40 Dec  8 18:12 /tmp/bla
$ touch /tmp/bla/file
touch: cannot touch '/tmp/bla/file': Permission denied
$ sudo !!
sudo touch /tmp/bla/file
$ ls /tmp/bla/file
ls: cannot access '/tmp/bla/file': Permission denied
$ ls /tmp/bla
ls: cannot access '/tmp/bla/file': Permission denied
file
```

In other words: I don't think you should ever use mode `0760` on a directory.

Reply:
Ok, updating